app/workers/bgs/submit_form674_job.rb

```diff
@@ -42,14 +42,27 @@ def valid_claim_data(saved_claim_id, vet_info)
       claim.formatted_674_data(vet_info)
     end
 
-    def send_confirmation_email(user)
-      return if user.va_profile_email.blank?
+    def send_confirmation_email(user_uuid, va_profile_email, first_name)
+      return if va_profile_email.blank?
 
       VANotify::ConfirmationEmail.send(
-        email_address: user.va_profile_email,
+        email_address: va_profile_email,
         template_id: Settings.vanotify.services.va_gov.template_id.form686c_confirmation_email,
-        first_name: user&.first_name&.upcase,
-        user_uuid_and_form_id: "#{user.uuid}_#{FORM_ID}"
+        first_name: first_name&.upcase,
+        user_uuid_and_form_id: "#{user_uuid}_#{FORM_ID}"
+      )
+    end
+
+    def generate_temp_user(vet_info)
+      OpenStruct.new(
+        first_name: vet_info['full_name']['first'],
```

Comment:
Just a quick sanity check: do we need to nil safe this?

Reply:
I don't necessarily see a situation where vet_info will ever be blank but that's a good question. Would you suggest raising an error if vet_info is blank? of vet_info['full_name']? The object is built in the dependent_service in `get_form_hash_686c`

Reply:
I'd say not being very familiar with the codebase yet to just follow precedence but maybe at some point take a look at if that ever happened? I see a few places in the existing code here where it blindly returns if data is missing, just a red flag to me generally about silent failures. So I'd say again not knowing much about the codebase yet to go with your gut on what's in scope for this project